repository/booking_queries.py

```diff
@@ -279,3 +274,10 @@ def find_not_used_and_not_cancelled_bookings_associated_to_outdated_stock() -> L
         .filter(Booking.isCancelled == False) \
         .filter(Stock.endDatetime + timedelta(hours=48) < datetime.utcnow()) \
         .all()
+
+
+def find_by_token(token: str) -> Booking:
```

Comment:
il y a déjà une fonction `find_by_validation_token` et ici il y a en plus un filtre par `isUsed == True`. Je pensais du coup plutôt à ce nom
```suggestion
def find_used_by_token(token: str) -> Booking:
```